resources/definitions/sovol_sv08.def.json

```
@@ -0,0 +1,37 @@
+{
+    "version": 2,
+    "name": "Sovol SV08",
+    "inherits": "voron2_base",
```

```suggestion
    "inherits": "voron2_base",
```
This type of inheritance (cross-manufacturer) is not recommended. This would mean that if anyone makes modification to voron2 printer series, this printer will also get affected without being in the scope of the developer.

---

SV08 is a Voron 2.4, Sovol Zero is some other Voron derivative (I don't know much about it), the rest are different designs.

I can move the voron2_base overrides into this printer and inherit directly from fdm, I think maybe that's the safest.

---

That seems to be the safest choice given the context of the printer, that way you're sure the profiles and test prints remain valid.

---

I hope to get this done this weekend; there are some completely unrelated projects that take priority. I guess I also need a rebase at some point.

---

You don't really need to compile or run from source for checking the definitions.
Assuming you're on linux/mac, just copy the files into `~/.config/cura/<version>` there are folders for definition, extruders, variants, ... and it adds the printer when you launch that version of cura into the list, so you can check things out using an appImage or a package.

Tests/Linters also will catch if any of the overrides are not correct on the repository.

---

Evidently using the precompiled versions now hits https://github.com/Ultimaker/Cura/issues/19837 for me, so that's a no-go, too.

I uploaded the patch, so we'll see what the linters do.